src/db.c

```diff
@@ -227,6 +227,7 @@ static const struct col_type_map mfi_cols_map[] =
     { "album_artist_sort",  mfi_offsetof(album_artist_sort),  DB_TYPE_STRING, DB_FIXUP_ALBUM_ARTIST_SORT },
     { "composer_sort",      mfi_offsetof(composer_sort),      DB_TYPE_STRING, DB_FIXUP_COMPOSER_SORT },
     { "channels",           mfi_offsetof(channels),           DB_TYPE_INT },
+    { "flag",               mfi_offsetof(flag),               DB_TYPE_INT },
```

Comment:
Not sure if the DB_FLAG_NO_ZERO might be relevant like it is for rating

Reply:
NO_ZERO would not allow the user to 'reset' the marked flag; ie they decide to leave the track as-is